python/example_code/bedrock-agent/README.md

```diff
@@ -144,6 +154,51 @@ python flows/list_flows.py
 The example first lists the flows in the current AWS Region. It
 then prompts for a flow ID, which you can get from the list of flows. Finally, the example lists the flow versions and flow aliases for the flow ID that you entered.
 
+#### Create and invoke a managed prompt
+
+This example shows you how to do the following:
+
+- Create a managed prompt
+- Create a version of the prompt
+- Invoke the prompt using the version
+- Update the prompt
+- Create a new version
+- Invoke the updated prompt
+- Clean up resources (optional)
+
+Start the example by running the following at a command prompt:
+
+```
+python prompts/scenario_get_started_with_prompts.py
+```
+
+By default, the example will clean up all resources it creates. If you want to keep the resources for further exploration, use the `--no-cleanup` flag:
+
+```
+python prompts/scenario_get_started_with_prompts.py --no-cleanup
+```
+
+You can also specify a different AWS region or model ID:
+
+```
+python prompts/scenario_get_started_with_prompts.py --region us-west-2 --model-id anthropic.claude-3-sonnet-20240229-v1:0
+```
+
+#### List prompts
+
+Shows how to list Amazon Bedrock managed prompts and versions of a prompt.
+
+Start the example by running the following at a command prompt:
+
+```
+python prompts/list_prompts.py
+```
+
+The example first lists the prompts in the current AWS Region. It
+then prompts for a prompt ID, which you can get from the list of prompts. Finally, the example lists the prompt versions for the prompt ID that you entered.
```

Comment:
When I ran this one, it didn't seem to do the part about letting me enter an ID, it just listed the prompts.

Reply:
The description is wrong. I update the readme.